aws-lc-fips-sys/builder/main.rs

```diff
@@ -476,6 +527,8 @@ fn main() {
         println!("cargo:libssl={}_ssl", prefix_string());
     }
 
+    println!("cargo:conf={}", OSSL_CONF_DEFINES.join(","));
```

Comment:
Where will this be used?

Reply:
The rust-openssl crate largely uses these for toggling APIs that should be available.

Reply:
BoringSSL emits these as well, I could see them being generally useful for integrators of the sys crates that won't to be able to inter-opt between the different offerings.